docker-compose.yml

```diff
@@ -1,5 +1,12 @@
+x-common-keys-core: &common-keys-core
```

Comment:
I don't think that we should define an anchor because we are only running one container, and it just makes the compose file unnecessarily more complex.

Reply:
I agree, but I think it only makes sense when there are more settings and I think the people who know how to use it are the ones who would create an anchor them self, so I'm not sure :p